ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml

```diff
@@ -90,7 +90,7 @@ releases:
   oldest-supported-credentials-request:
     release:
       channel: candidate
-      version: 4.10.0-fc.0
+      version: 4.10.0-fc.1
```

Comment:
openshift/cluster-network-operator#1283 landed after fc.1, and is [new in fc.2][1], so we should use fc.2 here if we accept the changes and want to raise the oldest-supported cred-request version.

[1]: https://amd64.ocp.releases.ci.openshift.org/releasestream/4-stable/release/4.10.0-fc.2